src/Compilers/Core/Portable/InternalUtilities/EnumerableExtensions.cs

```diff
@@ -384,6 +390,7 @@ public static IComparer<T> ToComparer<T>(this Comparison<T> comparison)
             return ImmutableDictionary.CreateRange(keyComparer, items);
         }
 
+#nullable disable // Transpose doesn't handle empty arrays. Needs to be updated as appropriate.
```

Comment:
Do we have a bug tracking this?

Reply:
There is no need for the bug. It's unlikely we can hit this in practice, and whoever gets to enabling nullable for this portion of the file will trivially figure it out.